datasets/us/ss_most_wanted/crawler.py

```diff
@@ -0,0 +1,77 @@
+from zavod import Context
+from zavod import helpers as h
+from xml.etree import ElementTree
+from normality import collapse_spaces
+
+
+def get_element_text(doc: ElementTree, xpath_value: str, to_remove=[], position=0):
+    element_tags = doc.xpath(xpath_value)
+
+    tag_list = []
+    for tag in element_tags:
+        try:
+            tag_list.append(tag.text_content())
+        except Exception:
```

Comment:
We can change this bare `Exception` to an `AttributeError` just to make it clear that we are handling the case where the `tag` doesn't have the `text_content` method. Another possibility without using exception would be to verify the type of the `tag` variable:

```
if isinstance(node, etree._ElementUnicodeResult):
      # Append the tag
elif isinstance(node, etree._Element):
       # Append the tag.text_content()
 ```